modules/nw-kuryr-cleanup.adoc

```diff
@@ -0,0 +1,280 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-kuryr-sdn.adoc
+
+:_content-type: PROCEDURE
+[id="nw-kuryr-cleanup_{context}"]
+= Cleaning up resources after migration
+
+After migration from the Kuryr network plugin to the OVN-Kubernetes network plugin completes, clean up the resources that Kuryr created previously. 
+
+.Prerequisites
+
+* You installed the {product-title} CLI, `oc`.
+* You installed a Python interpreter.
+* You installed the `openstacksdk` Python package.
+* You installed the `openstack` CLI.
+* You have access to the underlying {rh-openstack} cloud.
+* You can access the cluster as a user with the `cluster-admin` role.
+
+[NOTE]
+====
+At the time of writing when OCP16 is used, it includes ancient version of
```

Comment:
nit openstack 16.2?

which specific 16.2 version was used? 16.2.4? Was it with Red Hat openstack?

Reply:
RHEL 8.4 and RH OSP 16.2 snapshot from around end of March.

Reply:
```suggestion
At the time of writing when Red Hat OpenStack 16.2 is used, it includes ancient version of
```

Reply:
Is this resolved?